src/App.js

```diff
@@ -1,37 +0,0 @@
-import React from 'react'
```

Comment:
With x0 v5, routing is based on files in the `pages` directory, so this component is no longer necessary